cheesed/cheezoid.py

```diff
@@ -1,3 +1,19 @@
+"""
+Copyright 2015 Ilya Sukhanov
```

Comment:
Interesting, looks like there are several practices online:
 1. include only the repo owner's name
 1. include everyone's names in every file (yuck!)
 1. put a blanket statement about individual works being copyright of their respective owners (but seems this may require a [NOTICE file](https://svn.apache.org/repos/asf/openoffice/trunk/main/avmedia/source/java/Player.java))
 1. redirect to AUTHORS file (e.g., "Copyright all AUTHORS")
 1. keep source files free of license terms, and put everything in the root LICENSE + AUTHORS + NOTICE (optionally)

Anything works for me. :)